data/mods/DinoMod/dinosaur.json

```
@@ -544,12 +502,10 @@
     "dodge": 1,
     "armor_bash": 3,
     "armor_cut": 1,
-    "item_chance": 0,
     "luminance": 0,
     "hp": 120,
-    "special_freq": [ 0 ],
-    "death_function": "NORMAL",
-    "special_attack": [ "BOOMER" ],
+    "death_function": [ "NORMAL" ],
+    "special_attack": [ [ "BOOMER", 20 ] ],
```

```suggestion
    "special_attacks": [ [ "BOOMER", 20 ] ],
```
just checking that this is the correct one, as all your other ones use "special_attacks"